runtime/doc/ui.txt

```diff
@@ -197,6 +197,7 @@ the editor.
 
 	The following keys are deprecated:
 
+	undeprecate?
```

Comment:
Originally deprecated here: https://github.com/neovim/neovim/commit/fa4c2601000df2d792b0de865da3ac8c43ab723f. With this change the original reasoning no longer holds so undeprecate?